snakemake/persistence.py

```
@@ -180,28 +179,38 @@ def locked(self):
             logger.info(
                 "Error: Directory cannot be locked. This usually "
                 "means that another Snakemake instance is running on this directory. "
                 "Another possibility is that a previous run exited unexpectedly."
+                f"File(s) of interest include the following:\n {lock_conflict}"
             )
         yield
 
     @contextmanager
     def lock(self):
-        if self.locked:
+        lock_conflict = self.lock_conflicts(lock_conflict)
```

Ruff (0.8.2)

Undefined name `lock_conflict`

(F821)